### DISCUSSION TITLE
How to prepare Android for an eBPF programs

### DISCUSSION THREAD

**Discussion by kochia3** [yanivagman/BPFroid#5] [Q&A]
After we have compiled epbf program build with libbpfgo or similar toolset, how to prepare an Android device or an Emulator to run them?
I guess Emulator is easy way but still I failed to run ebpf programs

**Comment by yanivagman** [comment]
Which error message did you get?

**Comment by kochia3** [comment]
I forgot that I need to compile a kernel with specefic config
